pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/gates/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.20)
```

3.20 or 3.21 for cuQuantum based devices? 
```suggestion
cmake_minimum_required(VERSION 3.21)
```

---

3.21